app/decorators/solidus_stripe/active_merchant/stripe_payment_intents_gateway_decorator.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module ActiveMerchant::StripePaymentIntentsGatewayDecorator
+  def authorize(money, payment_method, options = {})
```

Comment:
can't we pass this option to AM via the authorize method in our StripePaymentMethod?

Reply:
I think that makes more sense, thanks!